package.json

```diff
@@ -16,7 +16,7 @@
     "watch-client-js": "./scripts/watch-client-js.sh",
     "watch-less": "./scripts/watch-less.sh",
     "debug": "npm run watch-less & npm run watch-client-js & npm run watch-server-js",
-    "lint": "eslint src test templates",
+    "lint": "eslint --fix src test templates",
```

Comment:
Please change this back - the linting command shouldn't have any side-effects - it should just check whether the code is clean as it is.